common/locales/en/quests.json

```diff
@@ -10,7 +10,13 @@
     "questDetails": "Quest Details",
     "invitations": "Invitations",
     "completed": "Completed!",
+    "rewardsAllParticipants": "Rewards for all Quest Participants",
+    "rewardsQuestOwner": "Additional Rewards for Quest Owner",
+    "questOwnerReceived": "The Quest Owner has also Received",
```

Comment:
Good catch, I was doing some pretty interesting git stash stuff and I must have missed this on one of the pops. I'll fix this when the rest is reviewed.